docs/index.md

```diff
@@ -1491,7 +1491,7 @@ A typical setup might look something like the following, where we call `mocha.se
   <head>
     <meta charset="utf-8" />
     <title>Mocha Tests</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
For some reason, something that injects or formats the snippets, adds the useless end tags. Not a big deal, it's just it's useless.

Reply:
Welcome to the prettier hate club...